src/app/components/dropdown/dropdown.ts

```diff
@@ -191,7 +191,7 @@ export class DropdownItem {
                 </span>
             </ng-container>
 
-            <div class="p-dropdown-trigger" role="button" aria-label="dropdown trigger" aria-haspopup="listbox" [attr.aria-expanded]="overlayVisible ?? false" [attr.data-pc-section]="'trigger'">
+            <div class="p-dropdown-trigger" role="button" aria-label="dropdown trigger" (mousedown)="$event.preventDefault()" aria-haspopup="listbox" [attr.aria-expanded]="overlayVisible ?? false" [attr.data-pc-section]="'trigger'">
```

Comment:
Instead of adding inline code, could you please add a new method onMouseDown and cancel the event inside.

`(mousedown)="onMouseDown($event)"`

Reply:
done, let me know if it needs anything else